cmd/wardenkms/wardenkms.go

```diff
@@ -29,11 +29,13 @@ type Config struct {
 	KeyringMnemonic string `env:"KEYRING_MNEMONIC, required"`
 	KeyringPassword string `env:"KEYRING_PASSWORD, required"`
 
-	BatchInterval time.Duration `env:"BATCH_INTERVAL, default=8s"`
-	BatchSize     int           `env:"BATCH_SIZE, default=7"`
-	GasLimit      uint64        `env:"GAS_LIMIT, default=400000"`
-	TxTimeout     time.Duration `env:"TX_TIMEOUT, default=120s"`
-	TxFee         int64         `env:"TX_FEE, default=400000"`
+	BatchInterval       time.Duration `env:"BATCH_INTERVAL, default=8s"`
+	BatchSize           int           `env:"BATCH_SIZE, default=7"`
+	GasLimit            uint64        `env:"GAS_LIMIT, default=400000"`
+	AutoEstimateGas     bool          `env:"AUTO_ESTIMATE_GAS, default=false"`
+	GasAdjustmentFactor float64       `env:"GAS_ADJUSTMENT_FACTOR, default=1.1"`
```

Comment:
```suggestion
	GasLimit            uint64        `env:"GAS_LIMIT, default=400000"`
	AutoEstimateGas     bool          `env:"AUTO_ESTIMATE_GAS, default=true"`
	GasAdjustmentFactor float64       `env:"GAS_ADJUSTMENT_FACTOR, default=1.2"`
```

i think it should be enabled by default